rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -150,7 +150,7 @@ def __next__(self):
             self.labels = np.zeros(self.count_batch, dtype="int32")
             self.loader.GetBBLabels(self.labels)
             # 1D bboxes array in a batch
-            self.bboxes = np.zeros((self.count_batch*4), dtype="float32")
+            self.bboxes = np.zeros((self.count_batch*4), dtype="float64")
```

Comment:
do we need to store the boxes in float64

Reply:
I'm returning the bboxes as double so needs float64 to store the boxes. I'll implement the change we discussed about return boxes as float32